### PR TITLE
(DOCSP-12292): Update iOS tutorial for beta.5

### PR DESCRIPTION
**Pull request by cbush**

### CODE REVIEW

File: swift-ios/Task Tracker/WelcomeViewController.swift [hunk 1]
```diff
@@ -150,31 +146,20 @@ class WelcomeViewController: UIViewController {
...
-
-                    self!.navigationController!.pushViewController(TasksViewController(project: nil, projectRealm: projectRealm), animated: true);
-
-                } else {
-                    // For the second phase of the tutorial, go to the Projects management page. 
-                    // This is where you can manage permissions and collaborators.
-                    self!.navigationController!.pushViewController(ProjectsViewController(), animated: true);
+
+                // Go directly to the Tasks page for the hardcoded project ID "My Project".
+                // This will use a common project and demonstrate sync.
```
Comment (ccho-mongodb) [inline swift-ios/Task Tracker/WelcomeViewController.swift:153]: Nitpick: could you clarify what "this" refers to? Is it a Task? The previous sentence talks about a project ID which I don't think is the subject that "will use a common project and demonstrate sync".

**Review by ccho-mongodb — APPROVED** [review]
LGTM, one nitpick.

File: swift-ios/Task Tracker/WelcomeViewController.swift (hunk 1 above)
Comment (cbush) [inline swift-ios/Task Tracker/WelcomeViewController.swift:153]:
```suggestion
                // This tutorial uses a common project to demonstrate sync.
```

Comment (cbush) [reply swift-ios/Task Tracker/WelcomeViewController.swift]: Good call. Updated.

**Pull request merged**